cairo/src/state.cairo

```diff
@@ -417,6 +414,30 @@ namespace Internals {
         return _copy_accounts(accounts_start + DictAccess.SIZE, accounts_end);
     }
 
+    // @notice Iterate through the accounts dict and finalize them
+    // @param accounts_start The dict start pointer
+    // @param accounts_end The dict end pointer
+    func _finalize_accounts{range_check_ptr, accounts: DictAccess*}(
+        accounts_start: DictAccess*, accounts_end: DictAccess*
+    ) {
+        if (accounts_start == accounts_end) {
+            return ();
+        }
+
+        if (accounts_start.new_value == 0) {
+            // If we do a dict_read on an unexisting account, `prev_value` and `new_value` are set to 0.
+            // However we expected pointers to model.Account, and casting 0 to model.Account* will
+            // cause a "Memory address must be relocatable" error.
```

Comment:
Comment needs fixing

Reply:
? what is unclear (same comment as in kakarot btw)

Reply:
mmh it seems I hallucinated an other comment. Sorry for that. It's clear